src/docs/Contributing/Principles/Glossary/index.mdx

```diff
@@ -8,33 +8,34 @@ order: 2
 
 ## Design system
 
-A set of **connected patterns** and **shared practices**, coherently organized to serve the purpose of a digital product. <br />
+A set of **connected patterns** and **shared practices**, coherently organized to serve the purpose of a digital product.<br />
 When we refer to the design system, we are talking about the all thing. Patterns, code, design files, principles and even the workflow and processes.
 
-## Styleguide
-
-A tool to capture, collect and share design patterns and guidelines for their usage.
-
-When we refer to the pattern library, we are talking about the documentation website. In our case, this is the static site generator powered by Gatsby and all his pages.
-
-## Component
+## Abstract
 
-A component is a reusable UI element created from a need, that defined clear purposes with a unique visual identity that adapts to different contexts.
+Abstract is a software like GitHub, used by the designers to collaborate on design files.
```

Comment:
@tiloyi You can remove Abstract. We stopped using it since we started to work with Figma 🙂